test/cb.test.js

```diff
@@ -457,6 +457,29 @@ describe('Couchbase test', () => {
         });
       });
 
+      it('update and creat should filter _cas', async() => {
+        const record = {
+          id: '0',
+          name: 'Charlie II',
+          age: 24,
+          _cas: 'somethings'
+        };
+        await Person.destroyById(record.id);
+        await Person.create(record);
+        const connection = await Person.getConnector()._connection;
+        const { value } = await connection.getAsync(record.id);
+        value.should.have.property('name', record.name);
+        value.should.not.have.property('_cas');
+        await Person.update({ id: record.id }, Object.assign({ _type: 'test' }, record));
```

Comment:
why do you assign the `_type`?

Reply:
I want to test `_type ` was filtered in update